subgraphs/_reference_/src/prices/config/mainnet.ts

```diff
@@ -152,6 +152,23 @@ class baxaOverride implements OracleConfig {
   }
 }
 
+// https://github.com/messari/subgraphs/issues/2329
+class delperOverride implements OracleConfig {
```

Comment:
class should be UpperCase https://google.github.io/styleguide/tsguide.html#identifiers